src/vivarium_ciff_sam/constants/data_values.py

```diff
@@ -61,7 +61,12 @@ class __Wasting(NamedTuple):
     MAM_TX_ALTERNATIVE_EFFICACY: float = 0.75
 
     # Incidence correction factor (total exit rate)
-    SAM_K: float = ('sam_incidence_correction', get_lognorm_from_quantiles(median=6.7, lower=5.3, upper=8.4))
+    SAM_K: Tuple = (
+        'sam_incidence_correction', get_lognorm_from_quantiles(median=6.7, lower=5.3, upper=8.4)
+    )
+    ALTERNATIVE_SAM_K: Tuple = (
```

Comment:
the research documentation linked mentioned bumping back the start of simulation run by 6 months to get a burn in period. is there a research reference for the "Adds a new scenario branch for an alternative sam-k value"?

Reply:
I'm not sure if there is. It was mentioned in slack and in meetings though.